config-connector/solutions/iam/kpt/service-account/README.md

```diff
@@ -0,0 +1,52 @@
+Service Account
+==================================================
+
+# NAME
+
+  service-account
+
+# SYNOPSIS
+
+  Config Connector compatible YAML files to create a service account in your desired project, and grant a specific member a role (default to iam.serviceAccountKeyAdmin) for accessing the service account that just created.
+
+# CONSUMPTION
+
+  Using [kpt](https://googlecontainertools.github.io/kpt/):
```

Comment:
Let's use text before the command to describe the command. For example, here I'd say something like "Fetch the \[kpt](https://googlecontainertools.github.io/kpt/) package of the solution".

Reply:
Updated. Thanks!